src/multi.jl

```diff
@@ -347,18 +381,16 @@ function valid_inequalities(m::PODNonlinearModel, discretization::Dict, λ::Dict
         return
     end
 
+    # Minimum Formulation with Boundary Cuts
```

Comment:
do we still need this - let me discuss with harsha and russell and get back to you on this - for now have it

Reply:
Okay. This is good for testing purposes.

Reply:
You can remove this function with its solver options.

Reply:
Okay. This is addressed in the most recent commit. Just a note, this option is removed in a later PR. We can do it here and I will rebase the other branches.